src/locales/en.json

```diff
@@ -71,5 +71,10 @@
 	"CERTIFICATE_TEMPLATE_CREATED": "Template created successfully",
 	"CERTIFICATE_ALREADY_EXISTS": "Template already exists",
 	"CERTIFICATE_NOT_FOUND": "Template not found",
-	"CERTIFICATE_UPDATED_SUCCESSFULLY": "Template updated successfully"
+	"CERTIFICATE_UPDATED_SUCCESSFULLY": "Template updated successfully",
+	"ONLY_PROJECT_CAN_BE_SUBMITTED_FOR_REVIEW": "Project can be submitted for review",
+	"PROJECT_OBJECTIVE_NOT_ADDED": "Project objective not added",
+	"SUB_TASK_NOT_FOUND": "Sub task not found",
```

Comment:
@ankit1shahu Is this point addressed
The adopter should be able to set further validations for the links being added at the org level or instance level. For e.g. in DIKSHA only the diksha links should be allowed and thus the link added should be checked while saving for the same.

Reply:
@ankit1shahu also 
See the text on top in the right section - "You can add combination of these resources: I- Learning Resource + Subtask; II- One observation; III- Only Learning Resource(s); IV- Only Subtasks"

subtask is not mandatory